README.md

```diff
@@ -4,6 +4,11 @@
 
 
 # Homebridge
+[![Website](https://img.shields.io/website?down_color=red&down_message=offline&up_color=green&up_message=online&url=https%3A%2F%2Fhomebridge.io)](https://homebridge.io)
+[![npm](https://img.shields.io/npm/v/homebridge?style=flat)](https://www.npmjs.com/package/homebridge)
+[![npm](https://img.shields.io/npm/dt/homebridge?style=flat)](https://www.npmjs.com/package/homebridge)
+[![GitHub issues](https://img.shields.io/github/issues/nfarina/homebridge?style=flat)](https://github.com/nfarina/homebridge/issues)
```

Comment:
And number of open issues, not sure what's the benefit to have it a a badge. Same for PRs.

Reply:
removed it.